### ISSUE TITLE
Incorrect description of remove list administration option

### ISSUE THREAD

**Issue by racke** [sympa-community/sympa#1059]
![test_Test_list_admin](https://user-images.githubusercontent.com/143678/102714379-6a5b5a00-42ce-11eb-99b9-c1b291802c1a.png)

> Completely removes the current list. 

That is terribly wrong. _Purging_ a list would have that affect. It is also a contradiction to the following sentence:

> Listmaster privileges are required to restore a list once it has been removed. 

Suggestion: Relabel the button as "Close list" and adjust the description accordingly.

**Comment by ikedas** [comment]
I think both "Remove" and "Close" are not so strange.  However, if we chose "Close", I suppose "Restore" may also be changed to appropriate antonym such as "Re-open".

**Comment by racke** [comment]
Yes, let's use "Close" and "Open" with the description explaining details. I will try to work on a pull request this year.
